src/vm/lib/metadata/agent.js

```diff
@@ -242,8 +303,9 @@ MetadataAgent.prototype.updateZone = function (zonename, callback) {
                 cb(false);
                 return;
             } else if (err) {
-                // fail open when we can't stat
-                log.error({err: err}, 'cannot fs.stat() ' + zonename);
+                // fail open (meaning: force reload) when we can't stat
+                log.error({err: err, zonename: zonename},
+                    'cannot fs.stat(), reloading');
                 cb(true);
```

Comment:
I just realize that this callback pattern seems different than the `callback(errObject, output...)` that we use elsewhere. Why do we need to return a boolean value as the first parameter of the callback here and not an error object?